### PR TITLE
Task 01

### PR DESCRIPTION
**Pull request by Tresanne**

### CODE REVIEW

**Comment by Tresanne** [comment]
@cheuschober Hi professor, Can you explain why I am failing the nosetests because I do not understand it.

**Comment by cheuschober** [comment]
@Tresanne, notice where you did your decimal conversion -- you did it on the outside which means that when it first performs its arithmetic it's just integers meaning it's flooring everything BEFORE it gets converted to decimal.

**Comment by Tresanne** [comment]
@cheuschober  What do you mean when you say I did it on the outside?

**Comment by cheuschober** [comment]
@Tresanne, order of operations is important -- a helpful exercise might be breaking apart your math from being a one-liner into only doing one operating at a time, eg:

    x = degrees - 32
    x *= 5
    x /= 9

And so on. At each stage, what is the value and what should it be? What data type do you expect at these stages.

**Comment by Tresanne** [comment]
@cheuschober ok Thanks. I think I understand now

File: task_01.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    Returns:
+        Returns value as a decimal in celsius
+
+    Examples:
+        >>> fahrenheit_to_celsius(212)
+        Decimal('100')
+    """
+    degrees = (decimal.Decimal(degrees) - 32) * 5 / (9)
+    return decimal.Decimal(degrees)
```
Comment (cheuschober) [inline task_01.py]: @Tresanne Good but this last conversion is unnecessary -- when decimals interact with other number types, like integers, they return decimals so this last conversion is totally unneeded.

File: task_01.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        degrees(int): The degree that will be used
+
+    Returns:
+        str: Returns value as a decimal in kelvin
+
+    Examples:
+        >>> celsius_to_kelvin(100)
+        Decimal('373.15')
+    """
+    degrees = decimal.Decimal(ABSOLUTE_DIFFERENCE + degrees)
```
Comment (cheuschober) [inline task_01.py]: @Tresanne Decimal comments apply here as well since ABSOLUTE_DIFFERENCE is already a decimal itself.

File: task_01.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    Returns:
+        Returns value as a decimal in kelvin
+
+    Examples:
+        >>> fahrenheit_to_kelvin(212)
+        Decimal('373.15')
+    """
+    degrees = (celsius_to_kelvin(fahrenheit_to_celsius(degrees)))
+    return decimal.Decimal(degrees)
```
Comment (cheuschober) [inline task_01.py]: @Tresanne Since the earlier functions already return decimals, you don't need to return them yourselves.

**Pull request merged**